apps/frontend/src/libs/components/select/styles.module.css

```diff
@@ -18,13 +18,24 @@
 	border-radius: 6px;
 }
 
+.control-small {
+	min-height: 32px;
+	padding: 4.5px 7px;
```

Comment:
round the value

Reply:
But this will cause component be 33px height instead of 32px. Or this is not a big deal? or should I use `height: 32px` here?

Reply:
I suppose it's not a big deal so I round it for now